docs/05_Mock_tests.md

```diff
@@ -0,0 +1,57 @@
+# Mock Tests
+
+A web crawler is by nature dependant on the external service which it is crawling.
+In order to have a reliable CI pipeline, this external dependency needs to be removed.
+For this there is a system in place to automatically save the requests made to a service and their responses.
+These can then be used in CI to reliably test changes made to the Extractor and not have test failures due to API changes on the side of the service.
```

Comment:
```suggestion
These can then be used in the CI to reliably test changes made to the Extractor and not have test failures due to API changes on the side of the service.
```

Reply:
If a "the" is added I would change it to `the CI pipeline`.
`the CI` sound weird

Thoughts?

Reply:
Agreed.

Reply:
```suggestion
These can then be used in the CI pipeline to reliably test changes made to the Extractor and not have test failures due to API changes on the side of the service.
```